vsphere/resource_vsphere_host_virtual_switch_test.go

```diff
@@ -187,6 +187,41 @@ func TestAccResourceVSphereHostVirtualSwitch_changeToStandby(t *testing.T) {
 	})
 }
 
+func TestAccResourceVSphereHostVirtualSwitch_import(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+			testAccResourceVSphereHostVirtualSwitchPreCheck(t)
+		},
+		Providers:    testAccProviders,
+		CheckDestroy: testAccResourceVSphereHostVirtualSwitchExists(false),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccResourceVSphereHostVirtualSwitchConfig(),
+				Check: resource.ComposeTestCheckFunc(
+					testAccResourceVSphereHostVirtualSwitchExists(true),
+				),
+			},
+			{
+				ResourceName:      "vsphere_host_virtual_switch.switch",
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateIdFunc: func(s *terraform.State) (string, error) {
```

Comment:
I'm not sure what I have to do in this section...

Reply:
That function just needs to return the string you would use when importing the resource using ```terraform import```. In this case, it is pretty straight forward since we're just using the `id`. The function below returns the `id` from the switch that was created in the first test step.
```
ImportStateIdFunc: func(s *terraform.State) (string, error) {
  vars, err := testClientVariablesForResource(s, fmt.Sprintf("vsphere_host_virtual_switch.%s", "switch"))
  if err != nil {
    return "", err
  } 
  return vars.resourceID, err
},
```

Reply:
Ah it's simpler than what I thought. :) Thanks